docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -188,6 +195,13 @@ to optimize how Determined interacts with PBS:
       using the :ref:`pbs section <cluster-configuration-slurm>` ``default_compute_resource_pool``
       or ``default_aux_resource_pool`` option.
 
+   -  If a PBS queue is not homogeneous, you may create a resource pool that provides homogenous
+      resources out of that queue using a custom resource pool. Configure a :ref:`resource pool
+      <cluster-resource-pools>` with ``provider_type: hpc``, specify the underlying PBS queue name
+      to receive the job and include a :ref:`task_container_defaults
+      <master-task-container-defaults>` section with the necessary ``psb`` options to select the
```

Comment:
knit - typo. should be "pbs"

Reply:
Good catch.  Grammerly doesn't help much with that :-)